<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>XSTAFMainWindow</class>
 <widget class="QMainWindow" name="XSTAFMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>718</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>XSTAF</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/icons/icons/server.png</normaloff>:/icons/icons/server.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0">
     <widget class="QTableView" name="DUTView">
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>718</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>STAF</string>
    </property>
    <addaction name="actionSettings"/>
    <addaction name="separator"/>
    <addaction name="actionCheckAndStartSTAF"/>
    <addaction name="actionRefresh"/>
   </widget>
   <widget class="QMenu" name="menuDUTs">
    <property name="title">
     <string>DUTs</string>
    </property>
    <addaction name="actionAddDUT"/>
    <addaction name="actionRemoveDUT"/>
   </widget>
   <widget class="QMenu" name="menuFile_2">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNewWorkSpace"/>
    <addaction name="actionOpenWorkSpace"/>
    <addaction name="actionSaveWorkSpace"/>
   </widget>
   <widget class="QMenu" name="menuReport">
    <property name="title">
     <string>Report</string>
    </property>
    <addaction name="actionGenerateReport"/>
   </widget>
   <addaction name="menuFile_2"/>
   <addaction name="menuFile"/>
   <addaction name="menuDUTs"/>
   <addaction name="menuReport"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QDockWidget" name="XSTAFLogDock">
   <property name="windowTitle">
    <string>XSTAF Log</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_2">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QTextEdit" name="XSTAFLogEdit"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionSettings"/>
   <addaction name="separator"/>
   <addaction name="actionNewWorkSpace"/>
   <addaction name="actionOpenWorkSpace"/>
   <addaction name="actionSaveWorkSpace"/>
   <addaction name="separator"/>
   <addaction name="actionAddDUT"/>
   <addaction name="actionRemoveDUT"/>
   <addaction name="separator"/>
   <addaction name="actionCheckAndStartSTAF"/>
   <addaction name="actionRefresh"/>
   <addaction name="separator"/>
   <addaction name="actionGenerateReport"/>
  </widget>
  <action name="actionSettings">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/icons/settings.png</normaloff>:/icons/icons/settings.png</iconset>
   </property>
   <property name="text">
    <string>settings</string>
   </property>
  </action>
  <action name="actionAddDUT">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/icons/add.png</normaloff>:/icons/icons/add.png</iconset>
   </property>
   <property name="text">
    <string>addDUT</string>
   </property>
  </action>
  <action name="actionRemoveDUT">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/icons/minus.png</normaloff>:/icons/icons/minus.png</iconset>
   </property>
   <property name="text">
    <string>removeDUT</string>
   </property>
  </action>
  <action name="actionRefresh">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/icons/refresh.png</normaloff>:/icons/icons/refresh.png</iconset>
   </property>
   <property name="text">
    <string>refresh</string>
   </property>
  </action>
  <action name="actionCheckAndStartSTAF">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/icons/connect.png</normaloff>:/icons/icons/connect.png</iconset>
   </property>
   <property name="text">
    <string>checkAndStartSTAF</string>
   </property>
   <property name="toolTip">
    <string>checkAndStartSTAF</string>
   </property>
  </action>
  <action name="actionNewWorkSpace">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/icons/new.png</normaloff>:/icons/icons/new.png</iconset>
   </property>
   <property name="text">
    <string>newWorkSpace</string>
   </property>
   <property name="toolTip">
    <string>newWorkSpace</string>
   </property>
  </action>
  <action name="actionOpenWorkSpace">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/icons/open.png</normaloff>:/icons/icons/open.png</iconset>
   </property>
   <property name="text">
    <string>openWorkSpace</string>
   </property>
   <property name="toolTip">
    <string>openWorkSpace</string>
   </property>
  </action>
  <action name="actionSaveWorkSpace">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/icons/save.png</normaloff>:/icons/icons/save.png</iconset>
   </property>
   <property name="text">
    <string>saveWorkSpace</string>
   </property>
   <property name="toolTip">
    <string>saveWorkSpace</string>
   </property>
  </action>
  <action name="actionGenerateReport">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/icons/report_generate.png</normaloff>:/icons/icons/report_generate.png</iconset>
   </property>
   <property name="text">
    <string>generateReport</string>
   </property>
   <property name="toolTip">
    <string>generateReport</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
